src/component/footer/Footer.jsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import "./style.css";
+import facebook from "./facebook.png";
+import twitter from "./twitter.png";
+import instagram from "./instagram.png";
+import linkedin from "./linkedin.png";
```

Comment:
Kindly move these images to the image folder.

Reply:
@t-bello7 The reason I used Vanilla CSS for the remainder was that the Design had some positioning  while being responsive, so I did that to make it more understandable for me and easy to fix for others in the future.